charts/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.16.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.12.12-procore-0.0.12
+version: 3.12.13-procore-0.0.13
```

```suggestion
version: 3.12.12-procore-0.0.13
```
the first semvar is for the upstream atlantis project and the second one is our versioning of our procore implementation

---

got it ! thanks a lot for correcting it Rachel .